app/models/spam_reviews.rb

```diff
@@ -0,0 +1,4 @@
+class SpamReview < ApplicationRecord
```

Comment:
This file can safely disappear. It's a duplicate of spam_review.rb.

Reply:
Done